cost.go

```diff
@@ -28,9 +28,9 @@ type CostError struct {
 	// Path represents the path to the schema node containing the expression.
 	// The path-generating implementation here is different than the one
 	// in Kubernetes and will generate slightly different-looking output.
-	Path string
+	Path string `json:"path"`
 	// Cost represents the cost of the expression. This is a unitless value.
-	Cost uint64
+	Cost uint64 `json:"cost"`
```

Comment:
Should the cost limit that was exceeded also be included?

Reply:
Makes sense, the limit probably won't be visible/known by end-users otherwise - fixed